packages/fiori/src/NavigationLayout.ts

```diff
@@ -32,8 +33,8 @@ import NavigationLayoutCss from "./generated/themes/NavigationLayout.css.js";
  * ### Responsive Behavior
  *
  * On desktop and tablet devices, the side navigation remains visible and can
- * be expanded or collapsed using the `sideCollapsed` property. On phone devices, the side navigation
- * is hidden by default but can be displayed using the same `sideCollapsed` property.
+ * be expanded or collapsed using the `mode` property. On phone devices, the side navigation
+ * is hidden by default but can be displayed using the same `mode` property.
  *
```

Comment:
On desktop and tablet devices, the side navigation is visible by default and can be expanded or collapsed using the `mode` property. On phone devices, the side navigation is hidden by default and can be displayed using the `mode` property.